src/index.html

```diff
@@ -24,16 +24,20 @@
 <body class="docs-app-background">
   <material-docs-app></material-docs-app>
   <script>
-    window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
-    ga('create', 'UA-8594346-24', 'auto');
-    ga('send', 'pageview');
+  // Note this is a customised version of the GA tracking snippet
+  // See the comments below for more info
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;
+  ~i.name.indexOf('NG_DEFER_BOOTSTRAP')|| // only load library if not running e2e tests
```

Comment:
e2e tests run on local host or other non-prod host, this causes all the GA pings to be ignored/filtered out automatically. In other words, it's typically not necessary to treat e2e tests differently.